frontend/public/components/markdown-view.tsx

```diff
@@ -11,6 +12,7 @@ const markdownConvert = (markdown) => {
     openLinksInNewWindow: true,
     strikethrough: true,
     emoji: true,
+    extensions: ['quickstart'],
```

Comment:
Is there a way we can add the highlight extension only in the quick starts so that it's not possible to be used by other parts of the application inadvertently.
Perhaps compose a `QuickStartsMarkdownView` which contributes the extra extension.

Reply:
done